2200055/최빈수구하기.py

```diff
@@ -0,0 +1,8 @@
+from collections import Counter
+T=int(input())
+for i in range (T):
+    case=int(input())
+    st_score=list(map(int,input().split()))
+    cnt_score=Counter(st_score).most_common()
+    k=sorted(cnt_score,key=lambda x: (x[1],x[0]),reverse=True)
+    print(f'#{case} {k[0][0]}')
```

Comment:
`Counter` 활용 잘해주셨네요! 👍 
```suggestion
    print(f'#{i + 1} {k[0][0]}')
```
마지막 테스트 케이스 번호 출력에 오타가 있는 것 같습니다 😄